lib/runtime-core/src/module.rs

```diff
@@ -77,6 +78,14 @@ pub struct ModuleInfo {
 
     /// Custom sections.
     pub custom_sections: HashMap<String, Vec<u8>>,
+
+    /// Flag controlling whether or not debug information for use in a debugger
+    /// will be generated.
+    pub generate_debug_info: bool,
+
+    #[serde(skip)]
+    /// Resource manager of debug information being used by a debugger.
+    pub debug_info_manager: jit_debug::JITCodeDebugInfoManager,
```

Comment:
Shouldn't this fields be optional depending on compilation features?

Reply:
Fair enough, I can make that happen and also exclude the jit_debug module from runtime-core, too.

Will need to double check that this shows up on `docs.rs` though because it's not a default feature and ideally it'd still be documented